<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="highlightLabel">
     <property name="text">
      <string>&amp;Highlight color:</string>
     </property>
     <property name="buddy">
      <cstring>highlightButton</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="highlightButton">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Preview height:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="previewHeightSpin">
     <property name="suffix">
      <string> pixels</string>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>160</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>40</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Path to DjView4:</string>
     </property>
     <property name="buddy">
      <cstring>pathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="pathEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>200</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="maxPageSpin">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Maximum pages:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="highlightRootCheck">
     <property name="text">
      <string>Highlight root shapes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>highlightButton</tabstop>
  <tabstop>highlightRootCheck</tabstop>
  <tabstop>previewHeightSpin</tabstop>
  <tabstop>maxPageSpin</tabstop>
  <tabstop>pathEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
